docs/openapi.yaml

```diff
@@ -3169,3 +3169,87 @@ paths:
               $ref: ./api/info/get-fee-rate-request.schema.json
             example:
               $ref: ./api/info/get-fee-rate-request.example.json
+
+  /extended/v1/events/address/{principal}:
```

Comment:
These two endpoints should be mixed into one general-purpose endpoint that allows you to filter for events by principal, tx_id, type, etc:
```
/extended/v1/tx/events
```